uPortal-security/uPortal-security-core/src/main/java/org/apereo/portal/security/provider/PersonManagerExtenderImpl.java

```diff
@@ -0,0 +1,52 @@
+package org.apereo.portal.security.provider;
+
+import io.jsonwebtoken.lang.Assert;
```

Comment:
This seems like a strange class to use for `Assert.notNull()`.  (Isn't there a Spring alternative?)